src/garage/tf/models/categorical_cnn_model.py

```diff
@@ -20,13 +20,11 @@ class CategoricalCNNModel(Model):
 
     Args:
         output_dim (int): Dimension of the network output.
-        filter_dims (tuple[int]): Dimension of the filters. For example,
-            (3, 5) means there are two convolutional layers. The filter
-            for first layer is of dimension (3 x 3) and the second one is of
-            dimension (5 x 5).
-        num_filters (tuple[int]): Number of filters. For example, (3, 32) means
-            there are two convolutional layers. The filter for the first layer
-            has 3 channels and the second one with 32 channels.
+        filters (tuple(tuple(tuple(int), int))): Dimension and number of
+            filters. For example, (((3, 5), 3), ((3, 3), 32)) means there are
```

Comment:
I think that it would be easier to read if this order were reversed, especially when formatted as a block, e.g.

```
filters = (
    (3, (3, 5)),   # 3 x (3, 5)
    (32, (3, 3)),  # 32 x (3, 3)
)
```

Reply:
That makes sense. Will do.